diff --git a/config.h b/config.h
index c159035..e21b041 100644
--- a/config.h
+++ b/config.h
@@ -3,7 +3,7 @@
 
 static int topbar = 1;                      /* -b  option; if 0, dmenu appears at bottom */
 static int fuzzy = 1;                       /* -F  option; if 0, dmenu doesn't use fuzzy matching */
-static int instant = 0;                     /* -n  option; if 1, selects matching item without the need to press enter */
+static int instant = 1;                     /* -n  option; if 1, selects matching item without the need to press enter */
 static int center = 1;                      /* -c  option; if 0, dmenu won't be centered on the screen */
 static int min_width = 500;                 /* minimum width when centered */
 /* -fn option overrides fonts[0]; default X11 font or font set */
@@ -20,13 +20,13 @@ char *colors[][2] = {
 	[SchemeNorm] = { "#bbbbbb", "#222222" },
 	[SchemeSel]  = { "#eeeeee", "#005577" },
 	[SchemeOut]  = { "#000000", "#00ffff" },
-	[SchemeBorder] = { "#000000", "#005577" },
+	[SchemeBorder] = { "#000000", "#cf4f88" },
 	[SchemeHp]   = { "#bbbbbb", "#333333" },
 };
 /* -l option; if nonzero, dmenu uses vertical list with given number of lines */
-static unsigned int lines      = 0;
+static unsigned int lines      = 5;
 /* -g option; if nonzero, dmenu uses a grid comprised of columns and lines */
-static unsigned int columns    = 0;
+static unsigned int columns    = 1;
 
 /*
  * Characters not considered part of a word while deleting words
@@ -35,5 +35,5 @@ static unsigned int columns    = 0;
 static const char worddelimiters[] = " ";
 
 /* Size of the window border */
-static unsigned int border_width = 0;
+static unsigned int border_width = 1;
 
diff --git a/dmenu b/dmenu
index fae741e..ea12503 100755
Binary files a/dmenu and b/dmenu differ
diff --git a/dmenu.o b/dmenu.o
index 53d1991..752b46a 100644
Binary files a/dmenu.o and b/dmenu.o differ
