diff --git a/config.h b/config.h
index f926bf4..ad3cd17 100644
--- a/config.h
+++ b/config.h
@@ -99,8 +99,8 @@ char *termname = "st-256color";
 unsigned int tabspaces = 8;
 
 /* bg opacity */
-float alpha = 0.8;
-float alphaUnfocused = 0.6;
+float alpha = 1.0;
+float alphaUnfocused = 1.0;
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
@@ -139,7 +139,7 @@ static const char *colorname[] = {
  * foreground, background, cursor, reverse cursor
  */
 unsigned int defaultbg = 0;
-unsigned int bg = 17, bgUnfocused = 16;
+unsigned int bg = 258, bgUnfocused = 258;
 unsigned int defaultfg = 259;
 unsigned int defaultcs = 256;
 unsigned int defaultrcs = 257;
